scopes/preview/preview/generate-link.ts

```diff
@@ -28,14 +28,16 @@ export function generateLink(
   let modulesLinks;
   if (mainModulesMap) {
     modulesLinks = Object.entries(mainModulesMap).map(([envId, path]) => {
-      const resolveFrom = toWindowsCompatiblePath(path);
+      // convert the path of files like mounters or docs templates into a path
+      // which starts from its package name
+      const resolveFrom = path.replace(/^.+\/node_modules\//, '');
```

Comment:
If I'm not wrong, this path is retrieved from the env config like `require.resolve('./preview/mounter.tsx')`. So it can't be a relative one from the original source. I didn't find any way better to do this yet.

Reply:
Maybe it will be better to calculate the relative path when creating the mainModulesMap like here - https://github.com/teambit/bit/blob/d3af25fd655666488ba2007b9901b403a4ce2ff9/scopes/preview/preview/preview.main.runtime.tsx#L618
(and also few lines below that point)
After calling the `previewDef.renderTemplatePathByEnv` 
you can do something like `path.relative(this.workspace.path, templatePath)`

Reply:
I moved the logic from `generate-link.ts` to `preview.main.runtime.tsx` as you mentioned. Now the e2e case was passed (locally). Thanks for the advice.

Reply:
For not using `relative()`, the reason is the same as not using `modulePath` in `preview.main.runtime.tsx` L644 as I commented.